### PR TITLE
Support recenttopics

### PR DESCRIPTION
**Pull request by avandenberghe**
please find a PR for a new event listener "recent_topics_mchat_side" to enable placing Mchat under recent topics when recent topics is set to "show at side". this event is available as of recent topics 2.2.3.

### CODE REVIEW

**Comment by avandenberghe** [comment]
The end result looks like : https://github.com/Sajaki/RecentTopics/issues/63

File: styles/prosilver/template/event/index_body_markforums_before.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<!-- IF S_RECENT_TOPICS && S_LOCATION_SIDE -->
+<!-- ELSE -->
 {% if MCHAT_INDEX and MCHAT_LOCATION %}
 	{% INCLUDE '@dmzx_mchat/mchat_body.html' %}
```
Comment (kasimi) [inline styles/prosilver/template/event/index_body_markforums_before.html:4]: Please add your two `S_`switches to the existing ìf` condition. Also, please don't mix old phpBB template syntax with Twig syntax and stick with Twig syntax only.

File: styles/prosilver/template/event/index_body_markforums_before.html [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
 {% endif %}
+<!-- ENDIF -->
```
Comment (kasimi) [inline styles/prosilver/template/event/index_body_markforums_before.html:6]: There's an empty new line missing at the end.

File: styles/prosilver/template/event/recenttopics_mchat_side.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<!-- IF S_RECENT_TOPICS && S_LOCATION_SIDE -->
+{% if MCHAT_INDEX and MCHAT_LOCATION %}
```
Comment (kasimi) [inline styles/prosilver/template/event/recenttopics_mchat_side.html:2]: Please add your two `S_` switches to the existing ìf` condition.

File: styles/prosilver/template/event/recenttopics_mchat_side.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+{% INCLUDE '@dmzx_mchat/mchat_body.html' %}
+{% endif %}
+<!-- ENDIF -->
```
Comment (kasimi) [inline styles/prosilver/template/event/recenttopics_mchat_side.html:5]: There's an empty new line missing at the end.

**Review by kasimi — CHANGES_REQUESTED** [review]
It looks like the code of the code of your other PR slipped into this PR as well. Can you please make a new one with only the changes for the Recent Topics extension?

**Pull request closed without merging**